app/controllers/requests_controller.rb

```diff
@@ -14,6 +14,9 @@ def new
   def create
     @request = Request.new(request_params)
     @request.user = current_user
+    content_array = @request.content.split(/[\r\n]/, 2)
```

Comment:
This is for separating the content to title and content.
First line (content_array[0]) becomes title, and rest (content_array[1]) become content.